distribution/docs/src/main/resources/_contents/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,845 @@
+
+==== CometD Endpoint
+
+The http://cometd.org[CometD] endpoint enables asychronous search capabilties.
+The CometD protocol is used to execute searches, retrieve results, and receive notifications.
+
+For an example of using CometD within a webapp see: `distribution/sdk/sample-cometd/`
+
+===== Installing CometD Endpoint
+
+The CometD Endpoint is installed by default with a standard installation in the ${ddf-ui} application.
+
+===== Configuring CometD Endpoint
+
+None.
```

Comment:
There are CometD configurations available under _Standard Search UI_ > _Search UI Endpoint_ that we probably should document here.